modules/python/init.zsh

```diff
@@ -46,6 +46,20 @@ if (( $? && $+commands[virtualenvwrapper.sh] )); then
   source "$commands[virtualenvwrapper.sh]"
 fi
 
+# Load PIP completion.
+if (( $+commands[pip] )); then
+  cache_file="${0:h}/cache.zsh"
```

Comment:
Good point, updating to use `$(mktemp -d)`. We'll have to take care of this in other places as well.